src/ec/packages/ec-component-search-form/docs/search-form.md

```diff
@@ -0,0 +1,40 @@
+# ECL Twig - EC Search Form
+
+npm package: `@ecl-twig/ec-component-search-form`
+
+```shell
+npm install --save @ecl-twig/ec-component-search-form
+```
+
+## Search Form
+
+### Parameters
+
+- "textInput" (associative array) default: A predefined structure for EC Text Input
```

Comment:
as we are using underscores (snake case) for all other twig variable names, we should use it there too.
(note that it only for twig variables, javascript variables are still in camel case)

Reply:
Fixed in https://github.com/ec-europa/ecl-twig/pull/28/commits/72483ba8d0c2d29290bdd9a64af43fa2ae0d59e7. Please check it.